doc/devel/index.rst

```diff
@@ -84,43 +84,63 @@ to contributing, we recommend that you first read our
    contribute
 
 .. grid:: 1 1 2 2
-   :class-row: sd-align-minor-center
+   :class-row: sd-fs-5 sd-align-minor-center
 
    .. grid-item::
-      :class: sd-fs-5
 
-      :octicon:`info;1em;sd-text-info` :ref:`Where should I start? <start-contributing>`
+      .. grid:: 1
+         :gutter: 1
+
+         .. grid-item::
+
+            :octicon:`info;1em;sd-text-info` :ref:`Where should I start? <start-contributing>`
+
+         .. grid-item::
+
+            :octicon:`question;1em;sd-text-info` :ref:`Where should I ask questions? <get_connected>`
 
-      :octicon:`question;1em;sd-text-info` :ref:`Where should I ask questions? <get_connected>`
+         .. grid-item::
 
-      :octicon:`git-pull-request;1em;sd-text-info` :ref:`How do I work on an issue? <managing_issues_prs>`
+            :octicon:`git-pull-request;1em;sd-text-info` :ref:`How do I work on an issue? <managing_issues_prs>`
 
-      :octicon:`codespaces;1em;sd-text-info` :ref:`How do I start a pull request? <how-to-pull-request>`
+         .. grid-item::
+
+            :octicon:`codespaces;1em;sd-text-info` :ref:`How do I start a pull request? <how-to-pull-request>`
 
    .. grid-item::
 
       .. grid:: 1
          :gutter: 1
-         :class-row: sd-fs-5
 
          .. grid-item-card::
             :link: contribute_code
             :link-type: ref
-            :shadow: none
+            :class-card: sd-outline-{primary} sd-shadow-none
+            :class-body: sd-text-{primary}
```

Comment:
These stylings seem to make the cards less responsive to mouse-over. There's only a minimal size/layout change:
https://output.circle-artifacts.com/output/job/5b25daa3-7646-458d-9b4d-a2b5ab7702fb/artifacts/0/doc/build/html/devel/index.html

Previously, at least the small frame changed to purple, indicating a link.
https://matplotlib.org/devdocs/devel/index.html